spec/spec_helper_acceptance_local.rb

```diff
@@ -0,0 +1,162 @@
+UNSUPPORTED_PLATFORMS = [].freeze
+
+def keytool_command
+  # The @keytool global does not exist right now as the function is defined.
+  # When the tests call the function, RSpec.shared_context below will have run
+  # by then and the variable will exist.
+  command = "\"#{@keytool_path}keytool\""
+  command.prepend('& ') if os[:family] == 'windows'
+  command
+end
+
+def create_and_upload_certs
+  cert_files = ['privkey.pem', 'ca.pem', 'ca2.pem', 'chain.pem', 'chain2.pem', 'leafkey.pem', 'leaf.pem', 'leafchain.pem', 'leafchain2.pem', 'leaf.p12', 'leaf2.p12']
+  recreate_certs = false
+  cert_files.each do |cert_file|
+    recreate_certs = true unless File.file?("spec/acceptance/certs/#{cert_file}")
+  end
+  create_certs if recreate_certs
+  cert_files.each do |cert_file|
+    if ENV['TARGET_HOST'].nil? || ENV['TARGET_HOST'] == 'localhost'
+      Open3.capture3("cp spec\\acceptance\\certs\\#{cert_file} C:\\tmp\\#{cert_file}")
```

Comment:
I think this is the line causing failures on Windows, Open3 defaults to cmd which does not have a `cp` command, we would need to use `copy` or drop into PowerShell.

This line also assumes the only time we're doing localhost testing is on a Windows machine?